lerna.json

```diff
@@ -3,6 +3,7 @@
   "packages": ["packages/*", "examples/*", "docs"],
   "command": {
     "publish": {
+      "forcePublish": "@loopback/cli,@loopback/docs",
```

Comment:
nitpick / style: can you put a space after the `comma`

Reply:
Like `"forcePublish": "@loopback/cli, @loopback/docs"`? I'm not sure if `lerna` allows so.

Reply:
You're right in that it won't handle a space after the `comma` BUT we can just set `forcePublish` to be an Array (or so is my understanding from this code: https://github.com/lerna/lerna/blob/master/utils/collect-updates/lib/get-forced-packages.js#L17-L20)